core/controllers/editor.py

```diff
@@ -137,6 +144,8 @@ def put(self, exploration_id):
         except utils.ValidationError as e:
             raise self.InvalidInputException(e)
 
+        are_changes_mergeable = exp_services.are_changes_mergeable(
```

Comment:
Can we call the variable `changes_are_mergable` to differentiate it from the function? Here and else where the variable is used. Let's do this in the HTTP response dicts too.

Reply:
But the ```changes_are_mergeable``` sound like that it is always true. This case is similar to the ```is_version_of_draft_valid``` where the function name and variable name both are the same, even in the HTTP response dicts. ```are_changes_mergable``` will store both true and false depending upon whether the changes are mergeable or not. That's why I kept this name. If it still needs to be changed then I'll update it.

Reply:
See note about boolean variables here: https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript

It is fine to call it changes_are_mergable and can represent true/false.

Reply:
Got it. Done.